runtime.ts

```diff
@@ -204,10 +211,16 @@ export class Runtime {
             this.host.exit(exitCode);
             return null;
           case OpCode.Jump:
-            this.notImplemented('Jump');
+            a = this.readShort();
```

Comment:
Remember, readShort increments PC as a side effect. This was such a dastardly bug!